src/pydocstyle/checker.py

```diff
@@ -284,10 +284,12 @@ def check_indent(self, definition, docstring):
                 indents = [leading_space(l) for l in lines if not is_blank(l)]
                 if set(' \t') == set(''.join(indents) + indent):
                     yield violations.D206()
-                if (len(indents) > 1 and min(indents[:-1]) > indent or
-                        indents[-1] > indent):
+                if (
+                    len(indents) > 1 and min(indents[:-1]) > indent or
+                    len(indents) > 0 and indents[-1] > indent
+                ):
                     yield violations.D208()
-                if min(indents) < indent:
+                if len(indents) > 0 and min(indents) < indent:
```

Comment:
replace len(indents) > 0

with if indents and....

Reply:
So, I wanted this to match the `> 1` above for clarity, it looks weirder not doing that -- I can change it if you insist but I felt it better to be consistent